doc/architecture/adr/0019-message-broker.md

```diff
@@ -0,0 +1,53 @@
+Message broker
+
+* Status:
+* Deciders: @anna-knudsen, @duizer, @ckr123, @Exobitt
+* Date: 2022-01-19
+
+---
+
+## Context and Problem Statement
+
+For our event driven architecture solution we need a message broker that can handle a large number of messages.
+We also want a message broker that is fairly easy to set up.
```

Comment:
I suggest that the Context states that this is for the Certificates domain specifically. ADR's are for the entire system, but in this case it is based on the needs within the Certificates domain.